test/evergreen.yml

```diff
@@ -1783,6 +1811,41 @@ tasks:
             set -o verbose
 
             ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1 2>&1
+  
+  - name: format-stress-sanitizer-test
+    #set a 25 hours timeout
+    exec_timeout_secs: 90000
+    depends_on:
+      - name: compile-sanitizer
+    commands:
+      - func: "fetch artifacts"
+        vars: 
+          dependent_task: compile-sanitizer
+      - func: "compile wiredtiger sanitizer"
+      - func: "format test script"
+        vars:
+          test_env_vars: ${test_env_vars|} ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0" ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer
```

Comment:
Remove assigning "test_env_vars" to itself ?

Reply:
Fixed